### PR TITLE
Review for Milestone 1

### PR DESCRIPTION
**Pull request by fabricio-garcia**
## Microverse requirements met

- [x] Use `create-react-app` to initialize a new React app
- Documentation: [https://github.com/facebook/create-react-app](https://github.com/facebook/create-react-app)
- [x] Delete any unnecessary file or asset. 
- [x] Add the necessary packages to your project:
- Add [`big.js`](https://github.com/MikeMcl/big.js/) using npm
- Add `prop-types` using npm for typechecking props.
- [X] Run the application with `npm start` and make sure that everything works correctly.
- [X] Follow the instructions in `create-react-app-buildpack` to deploy the app to Heroku
- Initialize a git repo
- Create a Heroku app
- Commit and deploy to Heroku
- Make sure the Heroku app works correctly
- Push everything to Github
- Documentation: [https://github.com/mars/create-react-app-buildpack](https://github.com/mars/create-react-app-buildpack)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "big.js": "^5.2.2",
```
Comment (bellom) [inline package.json:9]: I guess you missed this; You need to add `prop-types` 👍 

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/

File: README.md [hunk 2]
```diff
@@ -94,7 +92,7 @@ If you aren’t satisfied with the build tool and configuration choices, you can
 
 ## Live Demo
 
-[to-be-added-later](#)
+[React Calculator](https://react-calc-m.herokuapp.com/)
```
Comment (bellom) [inline README.md:95]: **Kindly update your `README` file and remove unnecessary content**.
Checkout out the below link for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

Comment (Oghenebrume50) [reply README.md:95]: @fabricio-garcia did you check out this comment here, your readme still has a lot of unnecessary detail, please remove them and try to put in very useful content following the provided template

**Comment by fabricio-garcia** [comment]
> ### STATUS: CHANGES REQUIRED
> 
> Hi @fabricio-garcia good job working on this project, everything is coming along fine, I have commented on what needed to be changed below in order to make this project start up properly and to meet basic requiremnets, please see comment below and make changes slightly_smiling_face
> **This is approved** +1
> 
> **RAPHAEL NORIODE** world_map

@Oghenebrume50 thank you for the review, I just checked the README for this branch and it is just fine. I believe you are reviewing the README from the master branch  which is incomplete.

**Pull request merged**